### ISSUE TITLE
Remove WIP notes

### ISSUE THREAD

**Issue by DinithiDiaz** [wso2/docs-mi#405]
## Description
Several pages in the documentation space are marked as WIP (Work in Progress). These notes had been added around 2020/2021 and are also present in the EI and APIM documentation. We need to remove the WIP notes after updating these pages with accurate content.

---

Examples:

[1] https://mi.docs.wso2.com/en/4.3.0/learn/examples/data-integration/distributed-trans-data-service/
[2] https://mi.docs.wso2.com/en/4.3.0/learn/examples/working-with-transactions/
[3] https://mi.docs.wso2.com/en/4.3.0/learn/examples/endpoint-examples/using-static-recepient-list-endpoints/

**Comment by DinithiDiaz** [comment]
**Update:**

Page [3] has already been reviewed, and the note was removed during the docs hackathon.

Page [1] was also reviewed as part of the docs hackathon effort but still contains the note. Therefore, I will confirm the accuracy of the content with Dedunu.

<img width="954" alt="wip-note" src="https://github.com/user-attachments/assets/349ff096-51cd-4292-bc4e-9467b99991c0">


Page [2] has not been reviewed yet, so we need to test the steps and verify the content's accuracy. Since it contains outdated information, we need to remove the page temporarily and test it before publishing again.

<img width="811" alt="wip-2" src="https://github.com/user-attachments/assets/a86bc24c-8669-4202-a66e-03194c3f18e2">

**Comment by DinithiDiaz** [comment]
The **How to Work with Transactions** page has been temporarily hidden from the live documentation space, and a redirection rule has been added (via https://github.com/wso2/docs-mi/pull/1022) because the content and images contain outdated information. We need to verify the accuracy of the content by testing the steps. Once this is tested, we need to update the page, uncomment the entry in the mkdocs.yml file, and remove the redirection. I have created [this issue](https://github.com/wso2/docs-mi/issues/1023) to track the requirement.

Therefore I'm closing this issue.
